### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stedankyi**
Fixed a typo in README.md

### Why
I was going through the readme for the course and came across this "mistake". I don't know if it is an Easter egg that the authors put inside this tutorial or something else so, I am trying to make this my first contribution on GITHUB.

### What's being changed:
I changed the sentence from :
"_With issues and pull requests, we see a more complete story **about about** the history, not just just the bare minimum_"
to :
"_With issues and pull requests, we see a more complete story **about** the history, not just just the bare minimum_"

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by sinsukehlab — APPROVED** [review]
I wonder why the words are repeated. :thinking:
4a555c1

**Comment by stedankyi** [comment]
Too much caffeine maybe?

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ When you want to create a crosslink, start typing the title of an issue or pull
 
 An important part of version control is the ability to look into the past. By using `git blame`, and finding the story behind a commit, we're able to do more than _blame_ people for code. We're able to see the story around why a commit was made. What is the associated pull request? Who approved the pull request? What tests were run on that commit before it was merged?
 
-The obvious reason to find things in history is to know about the history. With issues and pull requests, we see a more complete story about about the history, not just just the bare minimum.
+The obvious reason to find things in history is to know about the history. With issues and pull requests, we see a more complete story about the history, not just just the bare minimum.
```
Comment (sinsukehlab) [inline README.md:87]: Cf. #22
```suggestion
The obvious reason to find things in history is to know about the history. With issues and pull requests, we see a more complete story about the history, not just the bare minimum.
```

**Comment by sinsukehlab** [comment]
I will open a similar PR.

**Pull request closed without merging**